bin/companion.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Load local env vars. In CI, these are injected.
+if [ -f .env ]; then
+  # https://gist.github.com/mihow/9c7f559807069a03e302605691f85572?permalink_comment_id=3625310#gistcomment-3625310
+  set -a
+  source <(sed -e '/^#/d;/^\s*$/d' -e "s/'/'\\\''/g" -e "s/=\(.*\)/='\1'/g" .env)
```

Comment:
Have you considered https://github.com/motdotla/dotenv ?

Reply:
Yes, tried it here: https://github.com/transloadit/uppy/pull/3444/commits/11ede083297ef59b584e5f10a6a3a4b7eef5cf2a but couldn't get it to work in CI.